### ISSUE TITLE
About Loss

### ISSUE THREAD

**Issue by yqyao** [eriklindernoren/PyTorch-YOLOv3#7]
How about the training loss value? My implementation of YOLOv3 loss is very big and difficult to converge because the gird cell not containing the object is too large. Do you change something different from the original implementation?

**Comment by eriklindernoren** [comment]
I don't have an answer for why the loss is large for your implementation. I have followed implementations such as https://github.com/marvis/pytorch-yolo2 and the Darknet implementation and modified them for YOLOv3.
